docs/content/tutorial/step_03.ngdoc

```diff
@@ -127,7 +140,7 @@ acquired skill.
 <body>
 
   <!-- Use a custom component to render a list of phones -->
-  <phone-list></phone-list>
+  <phone-list></phone-list>       // This tells AngularJS to instantiate a phoneList component here.
```

Comment:
Since this is HTML, we should use HTML comments (`<!-- ... -->`).
I would also reduce the spacing (e.g. to 2 or 4 spaces).